File: Coding Projects/Section 10/RPGProject/Mage.cpp
//
// Created by dbehl on 5/14/2022.
//

#include "Mage.h"

Mage::Mage(string name, race Race) : Player(name, Race, 150,150) {
}

string Mage::attack() const {
    return "I will crush you with the power of my arcane missiles!";
}

File: Coding Projects/My Projects/delhi/delhi/Mage.cs

using System;
namespace rpg
{
	public class Mage : Player
	{
		public Mage(string name, race Race) : base(name,Race,150,150)
		{
		}
		override public string attack()
        {
			return "I will crush you with the power of my arcane missiles!";
        }
	}
}

